crates/zkvm/lib/src/ecdsa/affine.rs

```diff
@@ -0,0 +1,239 @@
+//! Implementation of the SP1 accelerated affine point.
+//!
+//! The [`crate::ecdsa::ProjectivePoint`] type is mainly used in the `ecdsa-core` algorithms,
+//! however, in some cases, the affine point is required.
+//!
+//! Note: SP1 uses affine arithmetic for all operations.
```

Comment:
```suggestion
//! Note: When performing curve operations, accelerated crates for SP1 use affine arithmetic instead of projective arithmetic for performance.
```